app/src/sharedTest/java/org/oppia/app/topic/play/TopicPlayFragmentTest.kt

```diff
@@ -25,13 +29,40 @@ import javax.inject.Singleton
 @RunWith(AndroidJUnit4::class)
 class TopicPlayFragmentTest {
 
+  // TODO(#137): Add following test-cases once story-progress function is implemented and expandable list is introduced.
+  //  Story progress is is displayed correctly.
+  //  Click on arrow to show and hide expandable list is working correctly.
+  //  Expandable list is showing correct chapter names.
+  //  Upon configuration change expanded list should remain expanded.
+  //  Click on story-title or entire item should open [StoryActivity].
+  //  Click on chapter in expandable list should start exploration.
+
+  @get:Rule
+  var activityTestRule: ActivityTestRule<TopicActivity> = ActivityTestRule(
+    TopicActivity::class.java, /* initialTouchMode= */ true, /* launchActivity= */ false
+  )
+
+  @Test
+  fun testTopicPlayFragment_loadFragmentWithTopicTestId0_storyName_isCorrect() {
+    ActivityScenario.launch(TopicActivity::class.java).use {
+      onView(atPosition(R.id.story_summary_recycler_view, 0)).check(matches(hasDescendant(withText("First Story"))))
+    }
+  }
+
   @Test
-  fun testTopicPlayFragment_loadFragment_textIsDisplayed() {
+  fun testTopicPlayFragment_loadFragmentWithTopicTestId0_chapterCount_isCorrect() {
     ActivityScenario.launch(TopicActivity::class.java).use {
-      onView(withId(R.id.dummy_text_view)).check(matches(withText("This is dummy TextView for testing")))
+      onView(atPosition(R.id.story_summary_recycler_view, 0)).check(matches(hasDescendant(withText("1 Chapter"))))
     }
   }
 
+  @Test
+  fun testTopicPlayFragment_loadFragmentWithTopicTestId0_configurationChange_storyName_isCorrect() {
```

Comment:
Isn't this verifying chapter count?

Reply:
Corrected this test and also introduced `containsString` everywhere in this.